.env.example

```diff
@@ -91,6 +91,11 @@ SPARK_TOKEN=myaccesstoken
 # CoreLightningWallet
 CORELIGHTNING_RPC="/home/bob/.lightning/bitcoin/lightning-rpc"
 
+# CoreLightningRestWallet
+CORELIGHTNING_REST_URL=http://127.0.0.1:8185/
+CORELIGHTNING_REST_MACAROON="/path/to/clnrest/access.macaroon of BASE64/HEX STRING"
```

Comment:
do not set an invalid path or base64 hexstring for the env variable, we had this issue before it can lead to invalid field inside DB.

either comment it out, or put a correct path and and a comment